README.md

```diff
@@ -104,6 +104,19 @@ Edit payload `0.0.1/full` should look like this:
 
 **Note: that you do not need to provide statement GUIDs or any hashes.**
 
+## Authentication
+
+In order to make requests they need to be authenticated and with an [edit token](https://www.mediawiki.org/wiki/Manual:Edit_token) supplied in the request payload.
+Authentication can either be done by using the [action api](https://www.mediawiki.org/wiki/API:Login) or OAuth.
+
+OAuth requires the [OAuth Extension](https://www.mediawiki.org/wiki/Extension:OAuth) to be installed and a OAuth consumer created via the `wiki/Special:OAuthConsumerRegistration/propose` page.
+
+Follow the instructions [here]((https://www.mediawiki.org/wiki/OAuth/Owner-only_consumers)) on how to create a OAuth owner-only consumer.
```

Comment:
There’s an extra set of `()` here, which apparently causes GitHub to not render this as a link at all?

Reply:
done